browser/.beforeprettier

```diff
@@ -133,7 +133,10 @@
 /src/layer/SplitPanesContext.ts
 /src/layer/tile/AutoFillMarkerSection.ts
 /src/layer/tile/CellCursorSection.ts
+/src/layer/tile/CellSelectionHandleSection.ts
```

Comment:
It was proposed to clarify folder structure a bit.
Maybe we could from now put new files related to canvas sections put into `src/canvas` ?
It will be shorter and will separate new files from leftovers from the leaflet.
It's no longer about separate tiles :)